connectors/http/graphql/element-templates/graphql-outbound-connector.json

```
@@ -0,0 +1,464 @@
+{
+  "$schema" : "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name" : "GraphQL Outbound Connector",
+  "id" : "io.camunda.connectors.GraphQL.v1",
+  "description" : "Execute GraphQL query",
+  "documentationRef" : "https://docs.camunda.io/docs/components/connectors/protocol/graphql/",
+  "version" : 5,
+  "category" : {
+    "id" : "connectors",
+    "name" : "Connectors"
+  },
+  "appliesTo" : [ "bpmn:Task" ],
+  "elementType" : {
+    "value" : "bpmn:ServiceTask"
+  },
+  "groups" : [ {
+    "id" : "authentication",
+    "label" : "Authentication"
+  }, {
+    "id" : "endpoint",
+    "label" : "HTTP Endpoint"
+  }, {
+    "id" : "graphql",
+    "label" : "GraphQL query"
+  }, {
+    "id" : "timeout",
+    "label" : "Connection timeout"
+  }, {
+    "id" : "output",
+    "label" : "Output mapping"
+  }, {
+    "id" : "error",
+    "label" : "Error handling"
+  }, {
+    "id" : "retries",
+    "label" : "Retries"
+  } ],
+  "properties" : [ {
+    "value" : "io.camunda:connector-graphql:1",
+    "binding" : {
+      "property" : "type",
+      "type" : "zeebe:taskDefinition"
+    },
+    "type" : "Hidden"
+  }, {
+    "id" : "authentication.type",
+    "label" : "Type",
+    "description" : "Choose the authentication type. Select 'None' if no authentication is necessary",
+    "value" : "noAuth",
+    "group" : "authentication",
+    "binding" : {
+      "name" : "authentication.type",
+      "type" : "zeebe:input"
+    },
+    "type" : "Dropdown",
+    "choices" : [ {
+      "name" : "API key",
+      "value" : "apiKey"
+    }, {
+      "name" : "Basic",
+      "value" : "basic"
+    }, {
+      "name" : "Bearer token",
+      "value" : "bearer"
+    }, {
+      "name" : "None",
+      "value" : "noAuth"
+    }, {
+      "name" : "OAuth 2.0",
+      "value" : "oauth-client-credentials-flow"
+    } ]
+  }, {
+    "id" : "authentication.apiKeyLocation",
+    "label" : "Api key location",
```

Here and others similar cases mentioned in the hybrid one.

Also, probably worth checking code-based generation for correct labels too.

```suggestion
    "label" : "API key location",
```

---

see above